crates/turbopack-ecmascript-runtime/js/src/shared-node/node-utils.ts

```diff
@@ -47,7 +48,17 @@ externalRequire.resolve = (
   return require.resolve(id, options);
 };
 
-async function loadWebAssemblyFromPath(
+async function compileWebAssemblyFromPath(
+  path: string
+): Promise<WebAssembly.Module> {
+  const { readFile } = require("fs/promises") as typeof import("fs/promises");
+
+  const buffer = await readFile(path);
```

Comment:
Nit: Can we use a streaming file read instead of a full buffer?

Reply:
Is that possible? WebAssembly seems to only supports a request (mirroring the Web api), not a file stream

Reply:
Yup, just need to create a response that wraps the stream:

```js
const stream = createReadStream("path/to/module.wasm");
const response = new Response(Readable.toWeb(stream), {
  headers: {
    "content-type": "application/wasm",
  },
});
const { instance } = await WebAssembly.instantiateStreaming(response);
```

Reply:
Ah nice, will use that